plugins/com.google.cloud.tools.eclipse.sdk/src/com/google/cloud/tools/eclipse/sdk/internal/CloudSdkModifyJob.java

```diff
@@ -54,11 +57,22 @@
   public CloudSdkModifyJob(String jobName, MessageConsoleStream consoleStream,
       ReadWriteLock cloudSdkLock) {
     super(jobName);
-    this.consoleStream = consoleStream;
+    this.consoleStream = consoleStream != null ? consoleStream : createNewMessageConsole();
     this.cloudSdkLock = cloudSdkLock;
     setRule(MUTEX_RULE);
   }
 
+  @VisibleForTesting
+  MessageConsoleStream createNewMessageConsole() {
+    final MessageConsole console = MessageConsoleUtilities.findOrCreateConsole(
```

Comment:
why final?

Reply:
Oops, a remnant of an anonymous class creation.